modules/sdk-coin-avaxp/src/lib/utxoEngine.ts

```diff
@@ -0,0 +1,117 @@
+import { DecodedUtxoObj, SECP256K1_Transfer_Output } from './iface';
+import { BN, Buffer as BufferAvax } from 'avalanche';
+import { Signature } from 'avalanche/dist/common';
+import utils from './utils';
+import { BuildTransactionError } from '@bitgo/sdk-core';
+import { StandardAmountInput, StandardTransferableInput } from 'avalanche/dist/common/input';
+
+export interface InputData {
+  amount: BN;
+  txidBuf: BufferAvax;
+  outputidx: BufferAvax;
+  signareIdx: number[];
+  signatures: Signature[];
+}
+
+/**
+ * Inputs can be controlled but outputs get reordered in transactions
+ * In order to make sure that the mapping is always correct we create an addressIndx which matches to the appropiate
+ * signatureIdx
+ * @param {TransferableInput[]} utxos as transaction ins.
```

Comment:
param type doesn't match the function signature

Reply:
https://github.com/BitGo/BitGoJS/blob/6d023175be2d689b599bbc8e62e71c713fa7934a/modules/sdk-coin-avaxp/src/lib/utxoEngine.ts#L20